### ISSUE TITLE
[Bug]: kvsWebRtcThreadpool does not seem to install with make install

### ISSUE THREAD

**Issue by Drise13** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#2049]
### Please confirm you have already done the following

- [X] I have searched the repository for related/existing bug reports
- [X] I have all the details the issue requires

### Describe the bug

libkvsWebRtcThreadpool.so is made but not installed anywhere when using make install.

### Expected Behavior

I expect libkvsWebRtcThreadpool.so to be installed to the target install directory or default (`/usr/local/bin/`) if not specified.

### Current Behavior

```
# make install
[  3%] Built target kvsWebRtcThreadpool
[ 64%] Built target kvsWebrtcClient
[ 75%] Built target kvsWebrtcSignalingClient
[ 79%] Built target kvsWebrtcClientMaster
[ 84%] Built target kvsWebrtcClientViewer
[ 87%] Built target discoverNatBehavior
[ 93%] Built target kvsWebrtcClientMasterGstSample
[100%] Built target kvsWebrtcClientViewerGstSample
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/lib/libkvsWebrtcClient.so
-- Set runtime path of "/usr/local/lib/libkvsWebrtcClient.so" to ""
-- Installing: /usr/local/lib/libkvsWebrtcSignalingClient.so
-- Set runtime path of "/usr/local/lib/libkvsWebrtcSignalingClient.so" to ""
-- Up-to-date: /usr/local/include
-- Up-to-date: /usr/local/include/com
-- Up-to-date: /usr/local/include/com/amazonaws
-- Up-to-date: /usr/local/include/com/amazonaws/kinesis
-- Up-to-date: /usr/local/include/com/amazonaws/kinesis/video
-- Up-to-date: /usr/local/include/com/amazonaws/kinesis/video/webrtcclient
-- Installing: /usr/local/include/com/amazonaws/kinesis/video/webrtcclient/Include.h
-- Installing: /usr/local/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h
-- Installing: /usr/local/include/com/amazonaws/kinesis/video/webrtcclient/NullableDefs.h
-- Installing: /usr/local/bin/kvsWebrtcClientMasterGstSample
-- Set runtime path of "/usr/local/bin/kvsWebrtcClientMasterGstSample" to ""
-- Installing: /usr/local/bin/kvsWebrtcClientViewerGstSample
-- Set runtime path of "/usr/local/bin/kvsWebrtcClientViewerGstSample" to ""
-- Installing: /usr/local/bin/kvsWebrtcClientMaster
-- Set runtime path of "/usr/local/bin/kvsWebrtcClientMaster" to ""
-- Installing: /usr/local/bin/kvsWebrtcClientViewer
-- Set runtime path of "/usr/local/bin/kvsWebrtcClientViewer" to ""
-- Installing: /usr/local/bin/discoverNatBehavior
-- Set runtime path of "/usr/local/bin/discoverNatBehavior" to ""
```

Missing the kvsWebRtcThreadpool install target.

### Reproduction Steps

```
git clone --recursive https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c.git -b v1.10.2 && \
mkdir -p /amazon-kinesis-video-streams-webrtc-sdk-c/build && \
cd /amazon-kinesis-video-streams-webrtc-sdk-c/build && \
cmake .. && \
make install -j$(nproc)
```

### WebRTC C SDK version being used

1.10.2

### Compiler and Version used

clang version 10.0.0-4ubuntu1  Target: x86_64-pc-linux-gnu Thread model: posix InstalledDir: /usr/bin

### Operating System and version

Ubuntu 20.04

### Platform being used

N/A?

**Comment by sirknightj** [comment]
Hi @Drise13,

We're aware of this issue and am looking into it. In the meantime, please try out this patch: [build-fixes-for-1.10.2.patch](https://github.com/user-attachments/files/16616252/build-fixes-for-1.10.2.patch) and let us know if it fixes the issue you encountered.

Thanks!

**Comment by Drise13** [comment]
Yes, this patch works.

**Comment by sirknightj** [comment]
This has been included as part of v1.11 release.
